apps_script/execute/execute.py

```diff
@@ -14,11 +14,7 @@
 
 # [START apps_script_execute]
 def main():
-    """Shows basic usage of the Apps Script API.
-
-    Creates a Apps Script API service object and uses it to call an
-    Apps Script function to print out a list of folders in the user's root
-    directory.
+    """Runs the sample.
```

Comment:
> Runs the sample.
Seems to be a less descriptive way of describing what the main method does. Why are we deleting the description?

Reply:
I was adding that information back in from the old samples (pre-GitHub-migration), but only at the end realized that you had moved this up to a file-level comment on each quickstart. It seemed duplicative to list it twice, on both the file and the main method. Some linters require that you have a docblock on the method though, hence my approach of a simple "Runs the thing" one.

Would you rather I remove the file-level one and keep the method-level one?

Reply:
Thanks for clarifying. Ideally a developer could copy/paste the method or whole file and still get a good comment.

I'd still say "Calls the Apps Script API" or something a bit more descriptive and less meta than "Runs the sample.". The comment "sample" doesn't make sense when this code is copied into another codebase.

That said, the change is LGTM'd and I'm OK either way.

Reply:
Every code sample already includes a "Calls the X API" already (just before the request is made) so that would be duplicative. Instead I've moved the file-level doc block to the main method, as it was before the GitHub migration. PTAL.